apps/verticalhome/js/pin_apps_default.json

```diff
@@ -0,0 +1,33 @@
+{"pin_apps_default":
+[
+  {"entry_point": "dialer",
+    "name": "Phone",
+    "manifestURL": "app://communications.gaiamobile.org/manifest.webapp",
+    "icon":   "app://communications.gaiamobile.org/dialer/style/icons/dialer_84.png",
```

Comment:
Does this work? Can the homescreen just read these images cross-origin like this?  As a general thing, you can't count on every app having a hardcoded 84 size icon. I think there is utility code in the existing homescreen for finding the most appropriate size icon for a given app. You'll probably need to use something like that.